lib/charms/opensearch/v0/opensearch_keystore.py

```diff
@@ -34,46 +34,25 @@ class OpenSearchKeystoreError(OpenSearchError):
     """Exception thrown when an opensearch keystore is invalid."""
 
 
+class OpenSearchKeystoreNotReadyYetError(OpenSearchKeystoreError):
+    """Exception thrown when the keystore is not ready yet."""
```

Comment:
This error is thrown when we try to reach out to the to reload the keys via API and it fails.